tests/data/settings_test.yaml

```diff
@@ -1,59 +0,0 @@
----
```

Comment:
The point of this test config was to disable tqdm display which messes with the output of some test runners in some consoles. I would like to keep this intact.

Reply:
Fixed with https://github.com/crocs-muni/sec-certs/pull/275/commits/4085c61d62e3e9d307ee8f05f3409d3cd10188ad

Reply:
This is cool but this also needs to be setup to be loaded in the tests. Can we create some global pytest fixture that loads these settings as config? The previous test infra had this in some test's setUp methods. And I guess the test settings may need to be updated with the new defaults and property names (if that was not already done) and only the progress bar setting should differ from the default config.

Reply:
> This is cool but this also needs to be setup to be loaded in the tests. Can we create some global pytest fixture that loads these settings as config? The previous test infra had this in some test's setUp methods.

You're right, I'll fix that.

> And I guess the test settings may need to be updated with the new defaults and property names (if that was not already done) and only the progress bar setting should differ from the default config.

This should be reflected.